src/index.html

```diff
@@ -7,12 +7,53 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylessheet"
+      href="/src/styles/index.css"
     />
```

Comment:
There seems to be a typo in the `rel` attribute value. It should be `stylesheet` instead of `stylessheet`. This is necessary for the browser to correctly identify and load the CSS file.